test/Microsoft.AspNet.Mvc.IntegrationTests/CollectionModelBinderIntegrationTest.cs

```diff
@@ -548,6 +548,7 @@ private class Person4
 
         private class Address4
```

Comment:
Does this class include any tests that'll hit `IsBindingRequired` violations now?